### ISSUE TITLE
Updated parent theme and site will not render past heading.

### ISSUE THREAD

**Issue by melanieceraso** [holger1411/understrap#512]
I can't for the life of me figure out why the site is no longer rendering.  I have a child theme set up with our customizations and the last time I have successfully updated the Understrap theme was version 0.6.5.  

Since then, while I am able to update the theme, once done all that renders is the heading until the navigation.  

https://apruvestaging.wpengine.com/

Some direction or help would be greatly appreciated.  I love Understrap and would like to be able to stay on top of my theme updates.

Thanks, in advance.

Melanie

**Comment by holger1411** [comment]
The main nav code and markup was changed till 0.7.
If you copy the navigation structure into your child theme you have to change it along with the new version.
Check the parent themes header.php file and comapre it with your child theme´s header.php.
Here is the latest version:
https://github.com/holger1411/understrap/blob/master/header.php

**Comment by melanieceraso** [comment]
Thank you @holger1411 for the direction.  I've begun restyling my menu with the new template.  

The one thing that I think is worth noting is that initially, the nav was not responsive as I transitioned from the sm to md viewports.  I did some digging and updated the code on line 36 from <nav class="navbar navbar-expand-md navbar-dark bg-dark"> to remove the "navbar-expand-md" and change it to "navbar-toggleable-sm" to display the menu items for md+ viewports. 

Thanks again.
